hotness/hotness_consumer.py

```diff
@@ -619,12 +619,15 @@ def _comment_on_bugzilla_with_template(
             projectid=project_id,
             dist_git_url=dist_git_url,
         )
+        versions = package.version if package.version in retrieved_versions else retrieved_versions
+        if len(versions) > 1:
+            versions = f"New versions of {package.name} available."
         notify_request = NotifyRequest(
             package=package,
             message=description,
             opts={
                 "bz_short_desc": self.short_desc_template
-                % dict(name=package.name, latest_upstream=package.version)
+                % dict(name=package.name, latest_upstream=versions)
```

Comment:
This will not work, because this is adding to template. It will either produce something like this "<package.name>-New versions of <package.name> available is available" for multiple versions.
You will need to instead replace `bz_short_desc` key in `opts` dict.

Reply:
I see on the tests, it was only first shot :)